index.js

```diff
@@ -5,6 +5,19 @@ function throwTypeError() {
 const endings = ['/', ':', '?', '#'];
 const starters = ['.', '/', '@'];
 
+getDomainFromUrl('test', {});
```

Comment:
I guess this isn't supposed to be here? :)

Reply:
oops. Removed ;)